recipes/kanidm-clients/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+# https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks
+MACOSX_SDK_VERSION:            # [osx and x86_64]
+  - '10.12'                    # [osx and x86_64]
```

Comment:
This is needed as otherwise `_getentropy` can't be linked against.

See https://github.com/conda-forge/py-sr25519-bindings-feedstock/issues/5 and https://github.com/conda-forge/ruff-feedstock/issues/165